src/environments/environment.ts

```diff
@@ -5,7 +5,7 @@
 export const environment = {
   production: false,
   apiUrl: 'http://localhost:3001/api/accounts',
-  apiUrlCred: 'http://localhost:3001/api/credentials'
+  apiUrlUser: 'http://localhost:3001/api/users'
```

Comment:
store in  `environment.ts` only `apiUrl` which should be `http://localhost:3001/api`